apps/rush-lib/src/logic/taskRunner/test/TaskRunner.test.ts

```diff
@@ -108,5 +108,24 @@ describe('TaskRunner', () => {
           checkConsoleOutput(terminalProvider);
         });
     });
+
+    it('printedStdoutAfterErrorWithEmptyStderr', () => {
+      taskRunner.addTask({
+        name: 'stdout only',
+        isIncrementalBuildAllowed: false,
+        execute: (writer: ITaskWriter) => {
+          writer.write('Step 1 of important task' + EOL);
+          writer.write('Oh noes' + EOL);
```

Comment:
Use professional wording

Reply:
Well, we're testing a tool that prints errors to stdout, which is utterly non-professional, I thought it would be more realistic this way :wink: . Okay, okay, will edit.